tools/css-api/appsettings.json

```diff
@@ -13,7 +13,7 @@
     "EndpointPath": "/api-docs/swagger/{0}/swagger.json"
   },
   "Keycloak": {
-    "ClientId": "89ff6cf4-3755-4329-adab-ccfb74052c97",
+    "ClientId": "cc93a52f-f2b6-400a-91ee-5a76e5521a4c",
```

Comment:
This may be different if you Keycloak updates your database instead of using the new configuration file.